modules/hosted-cluster-single-node-recovery.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assembly:
+//
+// * hcp-backup-restore-dr.adoc
+
+:_content-type: PROCEDURE
+[id="hosted-cluster-single-node-recovery_{context}"]
+= Recovering an etcd member for a hosted cluster
+
+If one etcd member of a 3-node cluster has corrupted data, it might fail. To recover the etcd member, complete the following steps.
```

Comment:
I think a better wording for the first sentence would be:
`An etcd member of a 3-node cluster might fail due to corrupted or missing data.`

Reply:
Thanks! I incorporated that revision.